fda-model/paireddata/CurveMetaData.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace paireddata
+{
+    public class CurveMetaData
+    {
+        public string XLabel { get; }
+        public string YLabel { get; }
+        public string Name { get; }
+        public string Description { get; }
+        public string Category { get; }
+        public bool IsNull { get; }
+        public CurveTypesEnum CurveType {get;}
+        public CurveMetaData()
+        {
+            CurveType = CurveTypesEnum.StrictlyMonotonicallyIncreasing;
```

Comment:
how will you specify curve type if the model consumer strictly interacts with uncertain paired data?